ci: understand timeout test flakes

I have an open issue running with Travis to investigate. Will report updates here.

Flakes always appear to be related to content we end up fetching from GitHub's CDN:

```
ok  	github.com/myitcv/govim	21.699s
--- FAIL: TestInstallScripts (316.44s)
    --- FAIL: TestInstallScripts/scripts (0.00s)
        --- FAIL: TestInstallScripts/scripts/install_pathogen (300.29s)
            testscript.go:290: 
                # Test the Pathogen install approach (300.265s)
                > [!net] skip
                > env HOME=$WORK/home
                > env GOPATH=$CURRENT_GOPATH
                > exec git clone -q $PLUGIN_PATH $HOME/.vim/bundle/govim
                [stderr]
                Note: checking out 'c72fe631bf545ad16e36132ed00b047f7ed08d4f'.
                
                You are in 'detached HEAD' state. You can look around, make experimental
                changes and commit them, and you can discard any commits you make in this
                state without impacting any branches by performing another checkout.
                
                If you want to create a new branch to retain commits you create, you may
                do so (now or later) by using -b with the checkout command again. Example:
                
                  git checkout -b <new-branch-name>
                
                > mkdir $HOME/.vim/autoload
                > mkdir $HOME/.vim/bundle
                > exec curl -LSso $HOME/.vim/autoload/pathogen.vim https://raw.githubusercontent.com/tpope/vim-pathogen/master/autoload/pathogen.vim
                [stderr]
                curl: (28) Operation timed out after 300180 milliseconds with 0 out of 0 bytes received
                [exit status 28]
                FAIL: testdatainstall/install_pathogen.txt:12: unexpected command failure
```

`curl` exit status 28 is a timeout:

https://curl.haxx.se/libcurl/c/libcurl-errors.html

--- a/_scripts/runDockerRun.sh
+++ b/_scripts/runDockerRun.sh
@@ -16,4 +16,4 @@ else
 	vimCmd="$VIM_COMMAND"
 fi
 
-docker run $proxy --env-file .docker_env_file -e "VIM_COMMAND=$vimCmd" -v $PWD:/home/$USER/govim -w /home/$USER/govim --rm govim ./_scripts/dockerRun.sh
+docker run $proxy --network host --env-file .docker_env_file -e "VIM_COMMAND=$vimCmd" -v $PWD:/home/$USER/govim -w /home/$USER/govim --rm govim ./_scripts/dockerRun.sh

